### DISCUSSION TITLE
A few issues that must be fixed this update

### DISCUSSION THREAD

**Discussion by exploitedtaken** [inspireui/support#5377] [Working on Features]
I am using Woocommerce Product Add-Ons plugin.

For example, on the application. Burger $34.90 + Fries and Drink $9.95 when added to the cart, the price is reflected in the total. However, when it comes to payment, the price of the extra option is not reflected. Only the $ 34.90 price of the product appears.

There is no such problem on the site. The extra price is reflected in the cart and I can order.

I'm just having this on the application side.

Another problem is in the variation part. I actively use the variation option in WCFM. eg. 1st burger in variation product. 2. Burger and drink options are active. There is no problem when I choose the options and add them to the cart. However, when I come to the payment section, I get a please select variations warning.

**Comment by hassanyousufx** [comment]
Yes, product add-ons are not reflected on the website. Please fix this issue urgently.

**Reply by ecomdeveloperbd** [reply]
Same, when I am using Woocommerce Product Add-Ons plugin

When you place an order from an app, that order information does not show up in the WordPress woocommerce order dashboard, but when you order from a website, it shows all the information correctly.

** Woocommerce Product Add -> 
Not all fields of this plugin work properly in apps. Especially the editbox field. Hopefully, this month's update on the mountain will solve this problem.

I hope you understand when you see the screenshot.

See the screenshot...

( working form buy when website )
screenshot: 1 : https://ibb.co/4VxJN7f

 ( do not work when from buy android apps )
screenshot: 2: https://ibb.co/LvkNXZm



this line add backend

/// To allow it, you can add this line to wp-config.php:
/// define('ALLOW_UNFILTERED_UPLOADS', true);

**Reply by hassanyousufx** [reply]
Does adding `define('ALLOW_UNFILTERED_UPLOADS', true);` fix this issue?

**Comment by exploitedtaken** [comment]
I tried the new update and Woocommerce Product Add-Ons plugin still not working properly.

/// To allow it, you can add this line to wp-config.php:
/// define('ALLOW_UNFILTERED_UPLOADS', true);

This detail has also been done. But the product options and price are not reflected on the payment page.

I need help :(

**Reply by ecomdeveloperbd** [reply]
Currently facing this error: 
woocommerce product addons, 
Not all input fields work properly. Multiple Choice, Quantity, Customer Defined Price, especially these fields are not working properly. If he gives multiple choice images, quantity, his show is not doing. But the website is doing exactly the show. It has to be solved

https://i.imgur.com/85cehHs.png

**Comment by exploitedtaken** [comment]
Although the product options are displayed on the order screen, if there is an extra amount in the product options section, that plus amount is not reflected on the payment screen. That's a big problem. This means loss on every order. I've been waiting for this issue to be fixed for the last 4 months :(

**Comment by hassanyousufx** [comment]
This issue is resolved in 2.8.0.

**Reply by exploitedtaken** [reply]
So how did you run it? Is webview payment enabled?

**Reply by hassanyousufx** [reply]
Yes, webview is enabled.

**Comment by exploitedtaken** [comment]
I tried, it still does not work, although the price of the extra product is reflected in the cart, but it is not reflected in the payment screen. The variation part in WCFM no longer works either. When you make a selection, it says no product, and no product is added to the cart. Still heartbreak :(

**Comment by minhcasi** [comment]
Hi, please update the latest 2.8.1 - we have upgraded the Webview feature to fix the product add-on - thank you!
